### DISCUSSION TITLE
False Positive with string "TU/e"

### DISCUSSION THREAD

**Discussion by DusterTheFirst** [crate-ci/typos#927] [General]
When using typos on my repository, the following file triggers the spell checker.

```md
---
name: TU/e
---
```

```
error: `Ue` should be `Use`, `Due`
  --> ./src/sponsors/platinum/TUe.md:2
```

It looks like it is ignoring the forward slash. I am unsure how to add "TU/e" to the exclude list. 

```toml
[default.extend-words]
"TU/e" = "TU/e"
```

makes no difference.

**Comment by epage** [comment]
If you run` typos --identifiers`, you'll see that we don't even consider `TU/e` a single identifier but two (which we then attempt to word split which you can see with `typos --words`).

You can use [`default.extend-ignore-re`](https://github.com/crate-ci/typos/blob/master/docs/reference.md#config-fields) to ignore arbitrary content.

**Reply by DusterTheFirst** [reply]
Both the `--identifiers` and `--words` functionality will be very useful for debugging/adding excludes. Thanks for the quick reply.
